src/app/sample/page.tsx

```diff
@@ -30,6 +32,30 @@ const Home: NextPage = () => {
       src: "/images/sample_chef.jpg",
     })),
   ];
+  // サーバーコンポーネントの場合
+  const getChef = async () => {
+    const data = await fetch("http://localhost:3000/api/chef");
+    // 型はtableType.tsにapiのパスと同じ名前の型があるのでそれを使用する
+    const chef: Chef = await data.json();
+    return chef;
+  };
+  // const Home: NextPage = () => {　とすると型エラーになるので、関数を定義してそこで非同期処理をしている
+  const chef = getChef();
+  console.log(chef);
```

Comment:
エラーハンドリング込みに変更したいです。
```suggestion
const res = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/api/chef`);
  if (!res.ok) {
    throw new Error("データの取得に失敗しました");
  }
  const chefs: Chef[] = await res.json();
```